cyber-resilience-sig/minutes/2025-06-09-mom-cyber-resilience-sig.md

```diff
@@ -28,9 +26,141 @@
 
 <details>
 <summary><b>Participants </b></summary>
- 
+
+* Tobie Langel (UnlockOpen/Eclipse Foundation)  
+* Juan Rico (Eclipse Foundation)  
+* Mathias Schindler (GitHub)  
+* Shanda Giacomoni (Eclipse Foundation)  
+* Dirk-Willem van Gulik (the Apache Software Foundation)  
+* Dick Brooks (Business Cyber Guardian)  
+* Becky Hepper (Seagate)  
+* Olle E. Johansson (Edvina / OWASP)  
+* Henry Haverinen (Cyberismo)  
+* Roman Zhukov (Red Hat)  
+* Alistair Woodman (Erlang Ecosystem System (EEF))  
+* Rebecca Rubul (Rust Foundation)  
+* Jakub Zelenka (PHP Foundation)  
+* Daniel Thompson (Tauri)  
+* Sal Kimmich (GadflyAI)  
+* Vicky Risk (ISC.org)  
+* Dan Appelquist (Samsung / OpenSSF Global Cybersec Policy WG)  
+* Salve J. Nilsen (CPANSec)  
+* Jeremy Stanley (Spec Committee, OpenInfra Foundation, SPI)  
+* CRob (OpenSSF)   
+* Simon Phipps (SWH/Meshed Insights)  
+* Æva Black
+
 </details>
 
+
+## Minutes
+
+### Welcome & approve previous minutes (Tobie)
+
+* [https://github.com/orcwg/orcwg/pull/103](https://github.com/orcwg/orcwg/pull/103)  
+* No objection. merging.	
+
+### Pending action items from last SIG call (Tobie)
+
+- [ ] Open PR to add statements on CVEs to deliverables plan & get SIG started on this.  
+  - No progress. This needs help  
+- [ ] Continue work on infrastructure facilitating coordination around standardization.  
+  - Not much progress.  
+- [x] Open pull request to create vulnerability handling task force and share on the mailing list: \#104  
+  - Waiting on the Github team being approved (see [orcwg/.eclipsefdn#6](https://github.com/orcwg/.eclipsefdn/pull/6)).
+  - [ ] Tobie to organize first vulnerability handling TF call
+  - Anyone missing, add your name below ([intital list)](https://github.com/orcwg/orcwg/blob/main/cyber-resilience-sig/minutes/2025-05-26-mom-cyber-resilience-sig.md#esos-updates):  
+      - Jeremy Stanley <jeremy@openinfra.dev>
+      - Sal kimmich <salkimmich@gmail.com>
+      - Olle E. Johansson <oej@edvina.net>
+      - Dick Brooks <dick@businesscyberguardian.com>
+      - Arnout Engelen <engelen@apache.org>
```

Comment:
@tobie Sorry missed adding myself there. Can you please modify it? Shouldn't this actually match list inhttps://github.com/orcwg/.eclipsefdn/pull/6/files

Reply:
Yeah, you're good.